hc/test.py

```diff
@@ -1,10 +1,10 @@
 from django.contrib.auth.models import User
-from django.test import TestCase
+from django.test import TransactionTestCase
 
 from hc.accounts.models import Member, Profile
 
 
-class BaseTestCase(TestCase):
+class BaseTestCase(TransactionTestCase):
```

Comment:
And the use here

Reply:
`TestCase` kept blocking the access to the database when we were testing sendalerts so we decided to use `TransactionTestCase` as it allowed for database concurrency. This way, we could assert that we were able to send alerts when the nag times were set in the database while also still running the other tests which also require database access.